lib/std/debug.zig

```diff
@@ -262,6 +262,12 @@ pub fn panicExtra(trace: ?*const builtin.StackTrace, first_trace_addr: ?usize, c
                 defer held.release();
 
                 const stderr = io.getStdErr().writer();
+                if (builtin.single_threaded) {
+                    stderr.print("main thread panicked: ", .{}) catch os.abort();
+                } else {
+                    const current_thread_id = std.Thread.getCurrentThreadId();
+                    stderr.print("thread {d} panicked: ", .{current_thread_id}) catch os.abort();
```

Comment:
```suggestion
                    stderr.print("thread {d} panic: ", .{current_thread_id}) catch os.abort();
```

(to match the other change)